src/TrackRowInfo.js

```diff
@@ -98,6 +98,24 @@ export default function TrackRowInfo(props) {
         const canvas = canvasRef.current;
         const teardown = draw(canvas);
         
+        // Handle mouse click interaction on each track
+        d3.select(canvas).on("click", () => {
```

Comment:
I think we will be able to move this into a child component once each vertical track has its own canvas (https://github.com/hms-dbmi/cistrome-higlass-wrapper/issues/59) but this looks great for now.

Reply:
Yes, that looks a great idea.